### ISSUE TITLE
Datetime validator and tooLate issue

### ISSUE THREAD

**Issue by jadengore** [ansman/validate.js#88]
I'm currently using a datetime validator to validate whether a user is `>=` 18 years old. I've been trying to wrap my head around how the `tooLate` option to the datetime validator works, and I am having some issues with my implementation.

1. `tooLate` is global to all datetime validators instead of specific to the set of conditions. Is there any way to make a specific condition that allows `tooLate` to be set per condition?

2. I want to be able to provide different errors for `tooLate` and a general format error. Currently, `options.message` will override `errors` if a message exists on a datetime condition. Is there any way to implement this?

**Comment by jrichard-km** [comment]
+1
I found a workaround to this issue, I change this code 
```js
// Before using it we must add the parse and format functions
// Here is a sample implementation using moment.js
validate.extend(validate.validators.datetime, {
  // The value is guaranteed not to be null or undefined but otherwise it
  // could be anything.
  parse: function(value, options) {
    return +moment.utc(value);
  },
  // Input is a unix timestamp
  format: function(value, options) {
    var format = options.dateOnly ? "YYYY-MM-DD" : "YYYY-MM-DD hh:mm:ss";
    return moment.utc(value).format(format);
  }
});
```

to


```js
// Before using it we must add the parse and format functions
// Here is a sample implementation using moment.js
validate.extend(validate.validators.datetime, {
  // The value is guaranteed not to be null or undefined but otherwise it
  // could be anything.
  parse: function(value, options) {
    if( options.tooEarly )
    {
        validate.validators.datetime.tooEarly = options.tooEarly;
    }
    if( options.tooLate )
    {
        validate.validators.datetime.tooLate = options.tooLate;
    }
    return +moment.utc(value);
  },
  // Input is a unix timestamp
  format: function(value, options) {
    var format = options.dateOnly ? "YYYY-MM-DD" : "YYYY-MM-DD hh:mm:ss";
    return moment.utc(value).format(format);
  }
});
```

**Comment by ansman** [comment]
This is now released in 0.10.0

**Comment by jadengore** [comment]
Thank you!
